src/sql/src/query_model/attribute/rejected_nulls.rs

```diff
@@ -0,0 +1,279 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! Defines the [`RejectedNulls`] attribute.
+//!
+//! The attribute value is a set of column references associated with
+//! each `QueryBox`. If any of the references is `NULL`, there is at
+//! least one predicate in that box that will evaluate to `FALSE for
+//! the corresponding row. For boxes without predicates, the
```

Comment:
Should this say "evaluate to FALSE or NULL"?

Nit: Add a \` to the end of \`FALSE.

That said, I feel like it would be clearer for this sentence to read something like, "If an input row has a NULL value for a column in the set, that row will be filtered away by at least one predicate in the box."